framework-api/framework-api-messaging/src/main/java/uk/gov/justice/services/messaging/provider/JsonEnvelopeProvider.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.services.messaging.provider;
```

Comment:
Convention seems to be to put this in a package that has "spi" in it, rather than "provider" - eg javax.json.spi